### DISCUSSION TITLE
What policies does phone link settings conflict with?

### DISCUSSION THREAD

**Discussion by goudekid** [Atlas-OS/Atlas#1295] [Help & Support] [answered]
Before using atlas, I used phone link extensively, but atlas disables this by default. It can be re-enabled via a script provided:

"By default, Atlas disables this Settings page as it conflicts with some policies. Run the Enable Mobile Device Settings.cmd script to enable it again."

I'd like to have phone link, but I also want to know exactly what policies it means here before I re-enable it.

**Comment by Ast3risk-ops** [comment, marked as answer]
> Before using atlas, I used phone link extensively, but atlas disables this by default. It can be re-enabled via a script provided:
> 
> "By default, Atlas disables this Settings page as it conflicts with some policies. Run the Enable Mobile Device Settings.cmd script to enable it again."
> 
> I'd like to have phone link, but I also want to know exactly what policies it means here before I re-enable it.

The .cmd file itself can be opened in a text editor. Just do that and you'll see exactly what it does.

You can also view the file [here](https://github.com/Atlas-OS/Atlas/blob/main/src/playbook/Executables/AtlasDesktop/3.%20General%20Configuration/Mobile%20Devices%20(Phone%20Link)/Enable%20Mobile%20Device%20Settings.cmd).

**Reply by VildAspen** [reply]
Hey, when i run the cmd file the mobile devices settings just open with all the settings greyed out with a yellow message on top *Some of these settings are hidden or managed by your organization*. In the command prompt i get this message:

'"C:\Windows\AtlasModules\Scripts\settingsPages.cmd"' is not recognized as an internal or external command,
operable program or batch file.
Finished, you can now use the mobile device settings to sync your phone.
Press any key to continue . . .

Any idea how i fix this?

**Reply by Ast3risk-ops** [reply]
> Hey, when i run the cmd file the mobile devices settings just open with all the settings greyed out with a yellow message on top _Some of these settings are hidden or managed by your organization_. In the command prompt i get this message:
> 
> '"C:\Windows\AtlasModules\Scripts\settingsPages.cmd"' is not recognized as an internal or external command, operable program or batch file. Finished, you can now use the mobile device settings to sync your phone. Press any key to continue . . .
> 
> Any idea how i fix this?

Is your Atlas version up to date (you can check in settings > system > about)? and does the `C:\Windows\AtlasModules` folder exist?
